static/js/movie.js

```diff
@@ -226,3 +226,56 @@ function renderNextPage(data) {
   const previousBtn = document.querySelector(".previous-btn");
   previousBtn.addEventListener("click", previousReviewPage);
 }
+
+// Send a review for the movie \\
+let rate = document.querySelector("#rate");
+let comment = document.querySelector("#addComment");
+let authorName = document.querySelector("#addName");
+let reviewBox = document.querySelector(".addReview");
+
+// Change temporary content of adding review box into Thank you text 
+function changeContent() {
+  var form = document.getElementById("form")
+  var originalContent = reviewBox.innerHTML;
+  reviewBox.innerHTML = "Tack för din recension!";
+  setTimeout(function() {
+    reviewBox.innerHTML = originalContent;
+  }, 10000) };
+
+document.querySelector("#addBtn").addEventListener("click", async (ev) => {
+  ev.preventDefault();
+
+  const body = {
+    author: authorName.value,
+    comment: comment.value,
+    rating: parseInt(rate.value),
+  }
+
+  const res = await fetch(`/api/reviews/${movieId}`, {
+    method: "POST",
+    mode: "cors",
+    credentials: "same-origin",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(body),
+  });
+
+  console.log(res.json())
+
+  document.querySelector("#rate").selectedIndex = 0;
+  document.querySelector("#addComment").value = "";
+  document.querySelector("#addName").value = "";
+
+  // Animation for the reviews
+  reviewContainer.style.height = `${reviewContainer.offsetHeight}px`;
+  reviewContainer.classList.add("screeningsAnim");
```

Comment:
I'm surprised to see `screeningsAnim` in the code for reviews. Is this reused code? Maybe abstract it by renaming it to something more generalized.